backend/src/app/database.py

```diff
@@ -2,6 +2,8 @@
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
 
+from .common.sample_contents import sample_contents_query
```

Comment:
```suggestion
from .query.sample_contents import sample_contents_query
```
쿼리가 계속 생긴다면 `query`로 모아도 될 것 같습니다

Reply:
넵. @ecpark4545 채도님도 같은 코멘트를 남겨주셨는데 나중에 반복이 생길때 모으면 좋을 것 같습니다 ㅎㅎ